governance/paper-process.md

```diff
@@ -0,0 +1,169 @@
+# Process for creating papers
+
+This document is intended to provide a consistent mechanism for cloud native
+security to produce community papers, ensure they are reviewed, and subsequently
+published.  Content of this document does not supercede existing processes and
+is intended to be used in conjuction with [existing proposal to project
+instructions](process.md).
+
+## Proposal
+
+If a proposal is made that includes a paper as a deliverable, the proposal needs
+to ensure that there is a clearly idenitified lead and a well defined paper
+scope.
+
+The paper scope and topic should be raised in at least one SIG meeting to
+solicit more volunteers (ideally 4).  Interested parties should meet at least
+once to describe the intent of the paper, & propose a very rough outline to
+present to SIG leadership for planning and scheduling as a project.
+
+## Project
+
+Once a SIG Leadership sponsor is assigned to the project, the group should meet
+to and agree on a tentative schedule.
+
+### Tentative schedule milestones
+
+The below list the minimum milestones that should be captured in a tentative
+schedule along with the estimated timeframes for completion.  Milestones are
+explained further in this document.
+
+| TODO | Milestone | Estimated time |
+| -- | -- | -- |
+| * [ ] | Audience, Goals, & refining scope | 1 week |
+| * [ ] | Tasking Assignment | 1 week |
+| * [ ] | Content Rough-in | 2-3 weeks |
+| * [ ] | Collaborative Review | 2 weeks |
+| * [ ] | Executive Summary and content wrap up | 2 weeks |
+| * [ ] | Narrative Voice | 1-2 weeks |
+| * [ ] | Final Group Review | 1 week |
+| * [ ] | Community Review | 2 weeks |
+| * [ ] | Public comment adjudication | 2 weeks (simultaneous with review) |
+| * [ ] | CNCF publishing engagement | ~2-3 weeks |
+| * [ ] | Addition to the repo | 2 weeks |
+| * [ ] | Blog post and publishing coordination | 1 week |
```

Comment:
I'd add more lead time for the blog article / aim to engage earlier with folks around publicity.

Reply:
```suggestion
| * [ ] | Blog post and publishing coordination | 2-3 weeks |
```